addons/udes_advanced_packaging/models/stock_quant.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models
+
+from collections import defaultdict
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.quant"
+
+    def group_quantity_by_product(self, only_available=False):
```

Comment:
Let's move this to `udes_stock`, this will be useful in other stock modules in the long term.